fetch-uj-records/splunk_container_default.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Why do we copy the `splunk_container_default.yaml` here instead of using the two templates available in splunk? 
take a look at `uid_map` and `ws_map`, they both use the kwok container template, but without duplicating it from the `kwok` directory.